src/Vivait/DelayedEventBundle/Job.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vivait\DelayedEventBundle;
+
+/**
+ * Allow job ID or other metadata to be used in process.
+ */
+final class Job
```

Comment:
What's the use case for this, is it so we can add it as a global log context or something? I think this implementation is fine, I'm not sure of another way of doing it either short of like setting values on a service or something, but not sure if that's overkill and I have a feeling there's cases where that wouldn't work

Reply:
It's so the job can poked during a long running process.